msvc/libmonoruntime-common.targets

```diff
@@ -86,6 +86,17 @@
     <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32handle.c" />
     <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32process.c" />
     <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32socket.c" />
+    
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\console-win32-uwp.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\console-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32error-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32event-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32file-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32mutex-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32process-win32-uwp.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32process-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32semaphore-win32.c" />
+    <ClCompile Include="$(MonoSourceLocation)\mono\metadata\w32socket-win32.c" />
   </ItemGroup>
```

Comment:
can we add a comment at the top of this block that we should remove these with pal versions